lib/ex_sdp.ex

```diff
@@ -68,5 +62,55 @@ defmodule ExSDP do
 
   defdelegate parse(text), to: Parser
   defdelegate parse!(text), to: Parser
-  defdelegate serialize(session), to: Serializer
+
+  @spec new(version :: non_neg_integer(), origin :: binary(), session_name :: binary()) :: t()
+  def new(version, origin, session_name) do
+    %__MODULE__{
+      version: version,
+      origin: origin,
+      session_name: session_name
+    }
+  end
+
+  @spec set_timing(sdp :: t(), timing :: Timing.t()) :: t()
+  def set_timing(sdp, timing) do
+    Bunch.Struct.put_in(sdp, :timing, timing)
+  end
```

Comment:
I'd avoid setters that don't involve any custom logic

Reply:
The intention was to be able to do something like this:
```elixir
    ExSDP.new(origin)
    |> set_timing(timing)
    |> add_attribute({"msid-semantic", " WMS 0YiRg3sIeAEZEhwD3ANvRbn7UFf3BjYBeANS"})
    |> add_attribute({"group", "BUNDLE 0 1"})
    |> add_media(get_audio_media(ice_ufrag, ice_pwd, fingerprint))
    |> add_media(get_video_media(ice_ufrag, ice_pwd, fingerprint))
    |> to_string()
```
Maybe we should add timing as parameter in `ExSDP.new()` function?

Reply:
It could be done like
```elixir
    %ExSDP{ExSDP.new(origin) | timing: timing})
    |> add_attribute({"msid-semantic", " WMS 0YiRg3sIeAEZEhwD3ANvRbn7UFf3BjYBeANS"})
    |> add_attribute({"group", "BUNDLE 0 1"})
    |> add_media(get_audio_media(ice_ufrag, ice_pwd, fingerprint))
    |> add_media(get_video_media(ice_ufrag, ice_pwd, fingerprint))
    |> to_string()
```